tests/exchange/test_hyperliquid.py

```diff
@@ -0,0 +1,319 @@
+            "entryPrice": 2521.9,
+            "side": "long",
+            "contracts": 0.0237,
+            "collateral": 19.902091,
+            "leverage": 3.0,
+            "liquidationPrice": 1699.14071943,
+        },
+        {
+            "symbol": "BTC/USDC:USDC",
+            "entryPrice": 68139.0,
+            "side": "short",
+            "contracts": 0.00145,
+            "collateral": 19.72573,
+            "leverage": 5.0,
+            "liquidationPrice": 80933.61590987,
+        },
+        {
+            "symbol": "SOL/USDC:USDC",
+            "entryPrice": 178.29,
+            "side": "short",
+            "contracts": 0.11,
+            "collateral": 19.605036,
+            "leverage": 1.0,
+            "liquidationPrice": 347.82205322,
+        },
+        {
+            "symbol": "SOL/USDC:USDC",
+            "entryPrice": 176.23,
+            "side": "long",
+            "contracts": 0.33,
+            "collateral": 19.364946,
+            "leverage": 3.0,
+            "liquidationPrice": 120.56240404,
+        },
+        {
+            "symbol": "SOL/USDC:USDC",
+            "entryPrice": 173.08,
+            "side": "short",
+            "contracts": 0.33,
+            "collateral": 19.01881,
+            "leverage": 3.0,
+            "liquidationPrice": 225.08561715,
+        },
+        {
+            "symbol": "BTC/USDC:USDC",
+            "entryPrice": 68240.0,
+            "side": "short",
+            "contracts": 0.00105,
+            "collateral": 17.887922,
+            "leverage": 4.0,
+            "liquidationPrice": 84431.79820839,
+        },
+        {
+            "symbol": "ETH/USDC:USDC",
+            "entryPrice": 2518.4,
+            "side": "short",
+            "contracts": 0.007,
+            "collateral": 17.62263,
+            "leverage": 1.0,
+            "liquidationPrice": 4986.05799151,
+        },
+        {
+            "symbol": "ETH/USDC:USDC",
+            "entryPrice": 2533.2,
+            "side": "long",
+            "contracts": 0.0347,
+            "collateral": 17.555195,
+            "leverage": 5.0,
+            "liquidationPrice": 2047.7642302,
+        },
+        {
+            "symbol": "DOGE/USDC:USDC",
+            "entryPrice": 0.13284,
+            "side": "long",
+            "contracts": 360.0,
+            "collateral": 15.943218,
+            "leverage": 3.0,
+            "liquidationPrice": 0.09082388,
+        },
+        {
+            "symbol": "SOL/USDC:USDC",
+            "entryPrice": 163.11,
+            "side": "short",
+            "contracts": 0.48,
+            "collateral": 15.650731,
+            "leverage": 5.0,
+            "liquidationPrice": 190.94213618,
+        },
+        {
+            "symbol": "BTC/USDC:USDC",
+            "entryPrice": 67141.0,
+            "side": "long",
+            "contracts": 0.00067,
+            "collateral": 14.979079,
+            "leverage": 3.0,
+            "liquidationPrice": 45236.52992613,
```

Comment:
Just to understand this right - these are positions you actually opened, - which we can then use as calculation reference, right?

Reply:
nevermind - i think https://github.com/freqtrade/freqtrade/pull/10839#discussion_r1821476969 actually covers this.